### ISSUE TITLE
Consistent Multivariate values for identities 

### ISSUE THREAD

**Issue by gagantrivedi** [SolidStateGroup/bullet-train-api#1882]
This should be identical across all the SDKs.
Needs to be implemented for:

- [x] Core [Uses Django ID or UUID]
- [x] Edge [Uses Identifier]
- [ ] Python [Uses Composite Key]
- [ ] Rust[Uses composite Key]
- [ ] Golang[Uses Identifier]
- [ ] Java[Uses Composite Key]
- [ ] Node [Django ID or composite Key]
- [ ] Edge proxy[Uses Identifier]
- [ ] Dot Net[Composite Key]
- [ ] PHP[Composite Key]
- [ ] Elixir[Hard to tell, I think composite key]

In order to avoid rolling v2 API's, For core, we will create a flag on environment named `use_mv_v2_evaluation` when turned on will use `composite key` else `identifier if dynamodb_enabled else (id or uuid))`, Edge will use that as flag as well to use `composite_key` if enabled instead of identifier. 
Release the Major version for Golang to use composite key.

gz#372

**Comment by matthewelwell** [comment]
@kyle-ssg this needs a small amount of FE work to expose the `use_mv_v2_evaluation` flag in the environment settings. We'll need to add some description around what it does to ensure people understand. The description should be something like: 


>Enabling this setting will ensure that multivariate evaluations made by the API are consistent with those made by local evaluation mode in our server side SDKs.
>
>Warning: toggling this setting will mean that some users will start receiving different multivariate values via the API / remote evaluation for any existing multivariate features that you have. Values received in local evaluation mode will not change.


@dabeeeenster happy with that description / warning?

**Comment by kyle-ssg** [comment]
This was marked as closed, though the feature flag it's associated with is disabled in production (mv_v2_setting), not sure if this is ready to enable based on the issue description

**Comment by matthewelwell** [comment]
@gagantrivedi can you confirm here?

**Comment by gagantrivedi** [comment]
Yeah, we can enable it - it is done for edge and core 
PS: I think node still needs to implement this behaviour

**Comment by matthewelwell** [comment]
> PS: I think node still needs to implement this behaviour

I don't think this is true since it currently uses: `identityModel.djangoID || identityModel.compositeKey`. Since the NodeJS SDK will never receive an identity that has a DjangoID, it will always use the compositeKey which is the desired behaviour. 

@gagantrivedi can you confirm that Go and Edge Proxy were done? They are not marked off in the list above.
